### PR TITLE
Add disconnected symbol and message for multiworld

### PR DESCRIPTION
**Pull request by ThanatosGit**
~~Draft:~~
1. ~~Need review on https://github.com/randovania/exlaunch/pull/5~~
2. ~~I can't get rid of the mip in bctex.~~
3. ~~Need a decision on which symbol to use finally~~
https://github.com/randovania/open-dread-rando/assets/117127188/2c5e86a7-568f-4fa6-b487-2de41cb429c7

### CODE REVIEW

File: src/open_dread_rando/files/custom_scenario.lua [hunk 1]
```diff
@@ -378,6 +391,8 @@ function Scenario.InitGui()
     if Init.bEnableRoomIds then
         RoomNameGui.Init()
     end
+
+    DisconnectGui.Init()
```
Comment (henriquegemignani) [inline src/open_dread_rando/files/custom_scenario.lua:395]: We are initializing the UI even when we're not interested in it.

**Review by henriquegemignani — CHANGES_REQUESTED** [review]

File: src/open_dread_rando/files/custom_scenario.lua (hunk 1 above)
Comment (ThanatosGit) [reply src/open_dread_rando/files/custom_scenario.lua]: It doesn't do any harm as the UI is not visible by default.

But I added a commit to not init the gui and also not schedule the function even once in no multiworld use case

Comment (henriquegemignani) [reply src/open_dread_rando/files/custom_scenario.lua]: We are wasting memory though! We keep using more and more and I'm a bit scared of when we run out

**Review by henriquegemignani — APPROVED** [review]
the image to use is up to you

**Comment by ThanatosGit** [comment]
I'm fine with the current one but how do I apply the MIT license for this icon correctly?
Do I need to add the license somewhere or I just name it in the sprite patch part? After searching for that I'm more confused than before and have no clue. (some said for Icons to just embed it into the metadata of the png but for us it's buried into the bctex without metadata etc.)

**Comment by ThanatosGit** [comment]
Changed the icon to the one from https://thenounproject.com/icon/unplug-3033613/
Added a folder with the original icon and a README with the link as it is provided when being logged in on the website.
See Video

https://github.com/randovania/open-dread-rando/assets/117127188/cdeca2be-6769-4bf9-8cde-1ca4e6cec9f1

 for new version

> General:
> You can find our general attribution rules[ here](https://help.thenounproject.com/hc/en-us/articles/200509928-How-do-I-give-creators-credit-in-my-work-).
> 
> You should follow the format Icon Name by Creator Name from Noun Project. Make sure to hyperlink the Noun Project portion where possible. It should link back to the browse page following these steps:
> 
> Utilize the link[ https://thenounproject.com/browse/icons/term/title-of-icon/](https://thenounproject.com/browse/icons/term/title-of-icon/)  and replace the “title-of-icon” portion for the icon’s title. For example, if you use an icon titled "Tree" you will adjust the hyperlink to [https://thenounproject.com/browse/icons/term/tree](https://thenounproject.com/browse/icons/term/tree?iconspage=1).

File: icons/README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Icons
+unplug by tezar tantular from <a href="https://thenounproject.com/browse/icons/term/unplug/" target="_blank" title="unplug Icons">Noun Project</a> (CC BY 3.0)
```
Comment (Miepee) [inline icons/README.md:2]: Why html link here instead of markdown link?
does `[https://blabla](Noun Project)` not work here?

Comment (ThanatosGit) [reply icons/README.md:2]: Honestly, haven't even looked at it. Just mindlessly pressed the button and CTRL+V
image

Comment (ThanatosGit) [reply icons/README.md]: Changed it.

**Comment by ThanatosGit** [comment]
Changed the original icon in the repo to the svg one which is much smaller. Rebased on main and force pushed.
If the it is OK with the `README.md` for crediting the author, it's done and could be merged.

**Comment by henriquegemignani** [comment]
> Changed the original icon in the repo to the svg one which is much smaller. Rebased on main and force pushed. If the it is OK with the `README.md` for crediting the author, it's done and could be merged.

Can you add a mention in the in-game credits?

**Comment by ThanatosGit** [comment]
Added the username to `Additional Art` which fits IMO.

**Pull request merged**